easybuild/easyconfigs/a/Automake/Automake-1.15-GCCcore-5.4.0.eb

```diff
@@ -1,15 +1,3 @@
-##
-# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
-#
-# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
-# Authors::   Fotis Georgatos <fotis@cern.ch>
-# License::   MIT/GPL
-# $Id$
-#
-# This work implements a part of the HPCBIOS project and is a component of the policy:
-# http://hpcbios.readthedocs.org/en/latest/
-##
-
```

Comment:
@vanzod Any reason why you trashed this? Better leave it there imho...

Reply:
My bad. I pushed the easyconfig file we use in our private repo where all non essential comments are stripped out. I'll fix it.